schema/omega-services.async.api.yaml

```diff
@@ -0,0 +1,442 @@
+      type: object
+      description: The properties of a record in an edit set.
+      required:
+        - ccr
+        - oci
+        - description
+        - coveringDate
+        - formerReferenceDepartment
+        - formerReferencePro
+        - startDateDay
+        - startDateMonth
+        - startDateYear
+        - endDateDay
+        - endDateMonth
+        - endDateYear
+        - legalStatusID
+        - placeOfDepositID
+        - note
+        - background
+        - custodialHistory
+        - relatedMaterial
+        - separatedMaterial
+        - creatorIDs
+
+      properties:
+        ccr:
+          description: |
+            CCR stands for "Classic Catalogue Reference"
+            The unique identifier which links the catalogue to the records it
+            represents and allows the user to order and refer to them.
+          $ref: "#/components/schemas/classicCatalogueReference"
+        oci:
+          $ref: "#/components/schemas/omegaCatalogueIdentifier"
+        description:
+          type: string
+          description: |
+            A prose statement giving information about the form and subject
+            matter of the unit of description (often in terms of subjects,
+            people and places). Its purpose is to enable users to judge the
+            potential relevance of the record described and whether they want
+            to investigate the material further or not.
+
+            Note: Called "Scope and Content" in TNA-CS13
+        coveringDate:
+          type: string
+          description: |
+            Identifies and records the date(s) of creation of the records being
+            described.
+        formerReferenceDepartment:
+          type: string
+          description: |
+            Record creating department's original reference for the record(s)
+            being described.
+        formerReferencePro:
+          type: string
+          description: |
+            Former PRO/TNA identifier providing the link between the archival
+            material and the description that represents it.
+        startDateDay:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        startDateMonth:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        startDateYear:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        endDateDay:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        endDateMonth:
+          type: string
+          description: |
+            Used for querying. Can be derived from Covering Dates or entered
+            manually.
+        endDateYear:
```

Comment:
The end date will be an ISO 8601 date, as you have already supposed in the ticket. The frontend will need to handle the rendering of this date into the form fields and parsing from the same.